packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/disabilityAssessments.js

```diff
@@ -367,6 +375,19 @@ const disabilityAssessments =  [
 			"medical condition, use %2$s instead. If you are not referencing the medical condition, consider other" +
 			" alternatives to describe the trait or behavior, such as %3$s.",
 	},
+	{
+		identifier: "OCD",
+		nonInclusivePhrases: [ "ocd" ],
+		inclusiveAlternatives: "<i>pedantic, obsessed, perfectionist</i>",
+		score: SCORES.POTENTIALLY_NON_INCLUSIVE,
+		feedbackFormat: "Avoid using <i>OCD</i>, unless talking about the specific medical condition. " +
+		"If you are not referencing the medical condition, consider other alternatives to describe the trait or behavior, such as %2$s. " +
+			"If you are referring to someone who has the medical condition, then state that they have OCD rather than that they are OCD.",
```

Comment:
After trying it myself, I understand why you're not using the `medicalCondition` variable as suggested in the issue description 😅 You want the first replacement variable to be replaced with "OCD" in uppercase, but we're not targeting it case-sensitively, because otherwise we would have to target the preceding forms case-sensitively as well. I would suggest doing it like this: 

```suggestion
		// We make sure to always capitalize "OCD" by pre-filling the first replacement variable.
		feedbackFormat: [ sprintf( medicalCondition, "OCD", "%2$s" ),
			"If you are referring to someone who has the medical condition, " +
			"then state that they have OCD rather than that they are OCD." ].join( " " ),
```

Of course you do need to import `sprintf` in that case, see the suggestion above.